knockoff/sdk/table.py

```diff
@@ -163,26 +263,27 @@ def copy_factory(self, columns=None, rename=None):
         rename = rename or {}
 
         def factory():
-            record = self.build_record()
+            record = self._build_record()
             out = []
             for col in columns:
                 out[rename.get(col, col)] = record[col]
             return out
+
         return factory
 
-    def build_record(self):
-        """TODO: should this be public?"""
+    def _build_record(self):
+        """
+        TODO: should this be public?
+              we can make this private and
+              use apply the column changes later
```

Comment:
Is there a typo here?

Reply:
yes, will remove that comment as _build_record was made private by convention